src/components/Social/ViewSocial.jsx

```diff
@@ -228,6 +230,103 @@ const MobileSocialIconsContainer = styled.div`
   }
 `
 
+const HeaderText = styled.h1`
+  font-size: 32px;
+  font-weight: 800;
+  margin-top: 1rem;
+
+  ${p => p.theme.mediaQueries.mobile} {
+    font-size: 28px;
```

Comment:
i think the header is left aligned on mobile!
```suggestion
    font-size: 28px;
    align-self: flex-start;
    padding-left: 5%;
```

| Figma                                                        | After                                                         |
| :------------------------------------------------------------ | :------------------------------------------------------------ |
| ![2025-03-02 at 10 40PM](https://github.com/user-attachments/assets/373df182-e73e-4d78-ba31-a6398f66e509) | ![2025-03-02 at 10 41PM](https://github.com/user-attachments/assets/2eda7d68-afab-48b7-9c06-01e3d0d5f3c6) |